pints/_log_likelihoods.py

```diff
@@ -237,6 +237,226 @@ def __call__(self, x):
         )
 
 
+class LinearHeteroscedasticGaussianLogLikelihood(
```

Comment:
Sorry, being dense -- why is it linear? Doesn't eta mean it can be non-linear?

Reply:
That is correct! Hmm I don't know why I thought it's linear. It somehow sounded need :D. Anyway it's not correct. Well nonlinear is probably to general. Let's just go back to ConstantAndMultiplicativeGaussianError then?

Reply:
done